site/sitemap.xml

```diff
@@ -672,27 +672,27 @@
   <changefreq>weekly</changefreq>
 </url>
 <url>
-  <loc>https://spark.apache.org/graphx/</loc>
+  <loc>https://spark.apache.org/sql/</loc>
```

Comment:
@foxish could you revert this part? it won't matter much either way. I think it's because people are using slightly different versions of jekyll or something (update to the latest if you're not already). This file keeps changing back and forth otherwise.

Reply:
Done